modules/cluster-and-ha-management/pages/remove-failed-node.adoc

```diff
@@ -52,7 +57,7 @@ NOTE: If you experience multi-node failures without modifying the default config
 You can check service status by running xref:system-management:management-with-gadmin.adoc#_gadmin_status[`gadmin status -v`].
 If a service is down, and you cannot restart it, please https://tigergraph.zendesk.com/hc/en-us/[open a support ticket].
 * There is at least 1 complete instance of the data.
-In other words, at least one replica does have a failed node.
+In other words, at least one replica does not have a failed node.
```

Comment:
"one partition", actually it would more clear if we say we need at lease one live replica for each partition.